src/installation/system_requirements.md

```diff
@@ -5,10 +5,10 @@ For the Epinio server, and related deployments we recommend to consider the foll
 - x86_64, ARM32, or ARM64 architecture
 - Supported operating systems
   - Linux
-    - k3s, k3d
-    - minikube (not yet)
+    - K3s, K3d
+    - minikube
   - Windows
-    - [Rancher Desktop](https://rancherdesktop.io)
+    - [Rancher Desktop](install_epinio_on_rancher_desktop.md)
   - MacOS
 - Kubernetes versions 1.18 .. 1.21
```

Comment:
This is not the scope of this PR but I think we need to update this too:
Feel free to commit it or I can do it in another PR.
```suggestion
- Kubernetes versions 1.20 .. 1.23
```